### PR TITLE
#860 - Update CODEOWNERS

### PR DESCRIPTION
**Pull request by obriensystems**
We can add anyone else from the team to these sections of the code in the CODEOWNERS list as well

### CODE REVIEW

**Review by tackaberry — DISMISSED** [review]

**Review by lucstjean-ssc — DISMISSED** [review]
/lgtm

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-*       @cartyc @fmichaelobrien
+*       @tackaberry @fmichaelobrien 
 
 # CLI Reviewers
 cli/*   @shaunmitchellve
 
 # Solution Reviewers
-solutions/* @davelanglois-ssc @alaincormier-ssc
\ No newline at end of file
+solutions/* @davelanglois-ssc @alaincormier-ssc
```
Comment (alaincormier-ssc) [inline .github/CODEOWNERS:9]: there's an old/outdated PR #281 to add some solutions reviewers
for now, we could add @lucstjean-ssc and @stephanemillaire-ssc

**Review by alaincormier-ssc — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (obriensystems) [reply .github/CODEOWNERS:9]: adding and closing #281

**Review by alaincormier-ssc — APPROVED** [review]

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

**Comment by fmichaelobrien** [comment]
adding my review in this special case - as a codeowner that is adding codeowners the already reviewed but by definition of this PR not in codeowners yet

**Review by fmichaelobrien — APPROVED** [review]

**Comment by fmichaelobrien** [comment]
getting green now - merging

**Pull request merged**